blocks/color-palette/style.scss

```diff
@@ -3,6 +3,11 @@ $color-palette-circle-spacing: 14px;
 
 .blocks-color-palette {
 	margin-right: -14px;
+
+	.blocks-color-palette__clear {
```

Comment:
Should we avoid nesting here? We tend to favor top-level rules when possible.

Reply:
Hi, the reason why this one is nested is for the CSS rule to have higher specificity. If the rule is not nested the margin is overwritten.